locales/en/plugin__odf-console.json

```
@@ -1153,6 +1153,25 @@
   "Share with presigned URL": "Share with presigned URL",
   "No objects found": "No objects found",
   "You do not have any objects in the bucket": "You do not have any objects in the bucket",
+  "Add Objects": "Add Objects",
+  "Drag and drop files/folders here.": "Drag and drop files/folders here.",
+  "Standard uploads have a size limit of up to 5GB in S3. For objects, multipart upload will upload the object in parts, which are assembled in the bucket.": "Standard uploads have a size limit of up to 5GB in S3. For objects, multipart upload will upload the object in parts, which are assembled in the bucket.",
```

It's better either to remove this statement to avoid confusion or make it 5 TB... AFAIK `UploadCommand` has a limit in TBs... `PutObjectCommand` has a 5 GB limit. 
```suggestion
 "For objects, multipart upload will upload the object in parts, which are assembled in the bucket.": "For objects, multipart upload will upload the object in parts, which are assembled in the bucket.",
```

---

This is currently as it stands in the mockups. Let's update the text when the microcopy is ready.